tests/foreman/cli/test_templatesync.py

```diff
@@ -126,3 +126,25 @@ def test_positive_export_filtered_templates_to_git(self):
 
         :CaseAutomation: NotAutomated
         """
+
+    @tier2
+    def test_positive_export_filtered_templates_to_temp_dir(
+        self, module_org, create_import_export_local_dir
```

Comment:
@jyejare is `create_import_export_local_dir` required for this test?

Reply:
@jameerpathan111 Not really, thanks for catching that. Now fixed !

Reply:
thanks, forgot to mention the other test, otherwise it LGTM.